CalibCalorimetry/CaloMiscalibTools/interface/EcalRecHitRecalib.h

```diff
@@ -44,13 +47,19 @@ class EcalRecHitRecalib : public edm::EDProducer {
 private:
   // ----------member data ---------------------------
 
-  std::string ecalHitsProducer_;
-  std::string barrelHits_;
-  std::string endcapHits_;
-  std::string RecalibBarrelHits_;
-  std::string RecalibEndcapHits_;
-  double refactor_;
-  double refactor_mean_;
+  const std::string ecalHitsProducer_;
+  const std::string barrelHits_;
+  const std::string endcapHits_;
+  const std::string RecalibBarrelHits_;
+  const std::string RecalibEndcapHits_;
```

Comment:
I think there is a CMS coding rule to have the members to start with lowercase letters

Reply:
There is a rule like that but the goal of the PR was mainly esConsumes migration rather than making the code conform to the code rules.

Reply:
Right, but since you touch the code, maybe it's a good opportunity to fix this? Would you rather do a follow-up cosmetics PR in the future?

Reply:
Done (commit follows).
Given that the last functional code change in this class seems to have been done 14 years ago by our current SP I guess it might actually not be in use anymore.